### PR TITLE
Project form prototype

### PR DESCRIPTION
**Pull request by velrest**
Depends on #1

### CODE REVIEW

File: addon/components/project-form/label.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class={{unless @noMargin "uk-margin"}}>
```
Comment (fkm-adfinis) [inline addon/components/project-form/label.hbs:1]: This too would have to come from `configured-classes`.

File: addon/components/project-form/date.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<PikadayInput
+  @format="DD.MM.YYYY"
+  @placeholder={{t "ember-gwr.projectForm.datePlaceholder"}}
+  @value={{@value}}
+  @onSelection={{@update}}
+  class="uk-width-1 {{configured-classes "date-picker"}}"
```
Comment (fkm-adfinis) [inline addon/components/project-form/date.hbs:6]: This `uk-width-1` and all others would have to come from the `configured-classes`.

File: addon/components/project-form/diff.hbs [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<div class={{configured-classes "diff-highlight"}}>
+  <div class="uk-margin">
+    <div class="uk-flex uk-flex-between uk-flex-bottom uk-margin-small-bottom">
```
Comment (fkm-adfinis) [inline addon/components/project-form/diff.hbs:3]: All these classes would have to come from the `configured-classes`.

File: addon/components/project-form/label.hbs [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  <label for={{@attr}} class={{if @required (configured-classes "text-bold")}}>
+    {{t (concat "ember-gwr.projectForm." @attr)}} {{if
+      @required
+      (t "ember-gwr.projectForm.required")
+    }}
+  </label>
+  {{yield}}
```
Comment (fkm-adfinis) [inline addon/components/project-form/label.hbs:8]: If you ever wanted to have a horizontal layout, you'd need an element wrapping around the yield.

* https://getuikit.com/docs/form#layout
* https://getbootstrap.com/docs/4.0/components/forms/#horizontal-form

I recommend adding that wrapper now. Even if horizontal forms are not currently planned.

File: addon/components/project-form/textarea.hbs [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<textarea
+  name="constructionProjectDescription"
```
Comment (fkm-adfinis) [inline addon/components/project-form/textarea.hbs:2]: This is probably an oversight.

File: addon/helpers/configured-classes.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+        if (Array.isArray(classesToAdd)) {
+          return classesToAdd.join(" ");
+        }
+
+        return classesToAdd;
+      })
+      .join(" ");
+  }
+}
```
Comment (czosel) [inline addon/helpers/configured-classes.js:20]: As discussed, let's drop this in a future PR.

**Review by czosel — APPROVED** [review]

**Pull request merged**